components/buttons/resolutionbutton.tsx

```diff
@@ -5,24 +5,38 @@ import Button from '@mui/material/Button';
 import { useTheme } from '@mui/material/styles';
 import { useAtom, useAtomValue } from 'jotai';
 import { halfResolutionAtom } from 'lib/atoms/atoms';
+import dynamic from 'next/dynamic';
 
 const ScaleIcon = () => {
     const halfResolution = useAtomValue(halfResolutionAtom);
     return halfResolution ? <Sd /> : <Hd />;
 };
 
-export default function ScaleButton() {
+const Resolution = dynamic(() => import('components/resolution'), {
```

Comment:
I've had troubles using `dynamic` before, it interacts weirdly with cloudflare pages somehow, will need to do more testing before merging.

Reply:
I recommend don't check on /new, something wrong with it and I beleave it's not my fault.
I fixed dynamic monaco in /new in this PR #332